submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+ 
+
+# **The Nuclear Options**
+
+<!--Draft 1.0, by Sir Fendi. Images are missing and some uncertainties about UPS are being researched. Also not sure about the appendix. All feedback is welcome about content, layout, or anything else. -->
+
+When I came across a glowing green patch of Uranium ore for the first time, I saw that Factorio had some juicy stuff waiting for me somewhere down the tech tree: Nuclear energy! While in today’s world the trials and triumphs of nuclear power are hotly debated, the Factorio community similarly has its ongoing discussions on whether it is worthwhile: Going nuclear is a couple steps up in complexity from the other power options, while also it is completely optional on the path to launching a rocket. Hence many players tend to not bother with diving into it, or just find a good blueprint as a blackbox and call it a day. Others hear of the warnings of the nuclear reactor as a UPS killer, or even a literal killer when it explode dramatically, and they decide to steer clear of it. Yes, there is a lot to look out for, but you can trust in Wube: Like many other carefully balanced mechanics of the game, nuclear power is reasonably easy to get into and you can quickly make something that just works. As for the more complicated and colorful part, I have a few guidelines to offer from my own fair share of tinkering and the findings of the glowing gems of our community.
+
+
+
+## Demystefying Nuclear Power
+
+While it may seem offputing with a whole bunch of new machines to it, at its core, a vanilla nuclear plant setup is only a little different from a burner steam power setup: You have a fuel being burned, heat creating steam, and an engine making pwoer from it. The extra complexity comes from three elements: Firstly, the fuel used in reactors is unique and is made in multiple steps instead of using the miner output directly. The official wiki has a [handy tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) that covers all the basics such as mining and centrifuging uranium ore, as well as setting up a basic single-reactor plant. 

```

Comment:
I would put the three points your making here as a numbered list:

1. Fuel is unique....
2. Do not stop burning fuel...
3. ....